### PR TITLE
refactor(var): move css var from root to class

### PR DESCRIPTION
**Pull request by YunYouJun**
- move css var from `:root` to `.el-xxx`
- use `sass` refactor `time-select`

---

Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/theme-chalk/src/popper.scss [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
+@import 'mixins/var';
 @import 'common/var';
 
-:root {
-  --el-popper-border-radius: var(--el-popover-border-radius, 4px);
+@include b(popper) {
+  @include set-component-css-var('popper', $--popper);
 }
 
 @include b(popper) {
```
Comment (JeremyWuuuuu) [inline packages/theme-chalk/src/popper.scss:9]: why not include it here?

File: packages/theme-chalk/src/radio.scss [hunk 2]
```diff
@@ -6,7 +6,7 @@
 @import 'mixins/button';
 @import 'common/var';
 
-:root {
+@include b(radio) {
```
Comment (JeremyWuuuuu) [inline packages/theme-chalk/src/radio.scss:9]: found a typo in line#15

Also can you move this into body after line#14?

**Review by jw-foss — APPROVED** [review]

**Comment by JeremyWuuuuu** [comment]
Found a typo, you might need to fix it before merging.

File: packages/theme-chalk/src/radio.scss (hunk 2 above)
Comment (YunYouJun) [reply packages/theme-chalk/src/radio.scss:9]: Fixed.

File: packages/theme-chalk/src/popper.scss (hunk 1 above)
Comment (YunYouJun) [reply packages/theme-chalk/src/popper.scss:9]: Compiled as a class that contains variables and a class that does not contain variables.

**Pull request merged**